### PR TITLE
Small cleanup of the web-wagmi example app

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: examples/web-wagmi/src/App.tsx [hunk 1]
```diff
@@ -68,7 +68,9 @@ const client = createClient({
 const lensConfig: LensConfig = {
   bindings: wagmiBindings(),
   environment: staging,
-  sources: [sources.lenster, sources.orb, 'any-other-app-id'],
+  sources: [],
+  // or narrow to the one you are interested in
+  // sources: [sources.lenster, sources.orb, 'any-other-app-id'],
```
Comment (desfero) [inline examples/web-wagmi/src/App.tsx:73]: can be confusing, especially when you compare the result with `lenster`.
Or when you add a comment and then go to the comments section and publication is not there given we don't add any `appId` in the metadata.

File: examples/web-wagmi/src/App.tsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ import { UseProfileFollowRevenue } from './revenue/UseProfileFollowRevenue';
 import { UseProfilePublicationRevenue } from './revenue/UseProfilePublicationRevenue';
 import { UsePublicationRevenue } from './revenue/UsePublicationRevenue';
 
-const { provider, webSocketProvider } = configureChains([polygon, optimism], [publicProvider()]);
+const { provider, webSocketProvider } = configureChains([polygon], [publicProvider()]);
```
Comment (desfero) [inline examples/web-wagmi/src/App.tsx:60]: `optimism` is a leftover

**Review by krzysu — APPROVED** [review]

File: examples/web-wagmi/src/App.tsx (hunk 2 above)
Comment (krzysu) [reply examples/web-wagmi/src/App.tsx:60]: shall we use `polygonMumbai` then?

Joaquin had similar finding in his PR https://github.com/lens-protocol/lens-sdk/pull/120/files

Comment (desfero) [reply examples/web-wagmi/src/App.tsx:60]: Good point, merging this one than and we will also keep the Joaquin changes in the #120

**Pull request merged**